cli/test/cli.ts

```diff
@@ -703,7 +703,7 @@ describe("CLI", () => {
                 assert.equal(log.args[0].length, 1);
 
                 var actual: string = log.args[0][0];
-                var expected: codePush.Package[] = [
+                var expected: cmdexec.PackageWithMetrics[] = [
```

Comment:
this and the changes above fix a typescript semantics bug we had that was preventing me from building